packages/datadog-plugin-moleculer/src/client.js

```diff
@@ -7,29 +7,37 @@ class MoleculerClientPlugin extends ClientPlugin {
   static get id () { return 'moleculer' }
   static get operation () { return 'call' }
 
-  start ({ actionName, opts }) {
+  bindStart (ctx) {
+    const { actionName, opts } = ctx
+
     const span = this.startSpan(this.operationName(), {
       service: this.config.service || this.serviceName(),
       resource: actionName,
       kind: 'client'
-    })
+    }, ctx)
 
     this.tracer.inject(span, 'text_map', opts.meta)
+
+    return ctx.currentStore
   }
 
-  finish ({ broker, ctx }) {
-    const span = this.activeSpan
+  bindFinish (ctx) {
```

Comment:
`bindFinish` is not needed since `runStores` is also not needed as described above, because promises already handle binding for us.